src/app/pages/office/cat-home-test/office.cat-home-test.page.html

```diff
@@ -101,22 +101,10 @@ <h4 class="fault-heading">ETA</h4>
                 </ion-card-content>
             </ion-card>
 
-            <ion-card id="eco" *ngIf="(pageState.ecoFaults$ | async) && (pageState.displayEco$ | async)">
-                <ion-card-content>
-                    <ion-grid>
-                        <ion-row class="mes-data ion-align-items-start">
-                            <ion-col size="32">
-                                <h4 class="fault-heading">ECO</h4>
-                            </ion-col>
-                            <ion-col class="ion-align-self-start ion-padding-start">
-                                <div id="ecoFaults">
-                                    {{pageState.ecoFaults$ | async}}
-                                </div>
-                            </ion-col>
-                        </ion-row>
-                    </ion-grid>
-                </ion-card-content>
-            </ion-card>
+            <eco-card
+                    [ecoDisplay]="pageState.displayEco$"
+                    [ecoFaults]="pageState.ecoFaults$"
+            ></eco-card>
 
```

Comment:
This should have an ```*ngIf``` around it like was there previously and then remove the if inside the component - the ```ecoDisplay``` input can then be completely removed